pkgs/development/python-modules/milksnake/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, buildPythonPackage, fetchurl, cffi }:
+
+buildPythonPackage rec {
+  pname = "milksnake";
+  version = "0.1.1";
+
+  src = fetchurl {
```

Comment:
Please use `fetchPypi`

Reply:
fetchPypi fails,maybe because it's a .zip and not a .tar.gz ? https://pypi.org/project/milksnake/#files

Reply:
found a fix

Reply:
You can pass `extension = "zip"` to `fetchPypi`.